packages/website/src/features/Packages/FunctionInput/DefaultInput.tsx

```diff
@@ -10,22 +9,7 @@ export const DefaultInput: FC<{
 }> = ({ handleUpdate, value = '', inputType }) => {
   const [updateValue, setUpdateValue] = useState<string>(value);
   useEffect(() => {
-    const bytes32Regex = /^0x[0-9a-fA-F]{64}$/;
-    if (inputType == 'bytes32' && !bytes32Regex.test(updateValue)) {
-      if (updateValue.startsWith('0x')) {
-        handleUpdate(updateValue);
-      } else {
-        handleUpdate(stringToHex(updateValue, { size: 32 }));
-      }
-    } else if (inputType == 'tuple') {
```

Comment:
want to make sure tuple inputs work on other parts of the site, will do some testing. otherwise, this is good to see!

Reply:
When I conformed if this affects any other places, this component is called on only one place which is FunctionInput.tsx and it has the condition of "tuple". So "tuple" function on DefaultInput.tsx is never used and we can remove it.